### PR TITLE
docs: add authoring tutorial

### PR DESCRIPTION
**Pull request by gvauter**
## Description

Adds a new tutorial for our documentation site that walks through the authoring process.  The content builds upon the existing [Github tutorial](https://redhatproductsecurity.github.io/trestle-bot/tutorials/github/).

Fixes # (issue)

## Type of change

- [X] This change requires a documentation update

### CODE REVIEW

File: docs/tutorials/authoring.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+4. Click the `Commit changes..` button
+5. Select the `Create a new branch for this commit and start a pull request` radio buton
+6. Click `Propose changes`
+
+
+The `Open a pull request` page now opens.  Enter any additional details about your changes into the description box.
+
+7. Click `Create pull request`
+8. For demo purposes, we will go ahead and merge the pull request ourselves.  Click `Merge pull request` and then `Confirm merge`.
```
Comment (jpower432) [inline docs/tutorials/authoring.md:32]: Perhaps here is would be beneficial to go into what would happen in a typical scenario as well

File: docs/tutorials/authoring.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+
+## Autosync
+
+Once the pull request has been merged the `Trestle-bot rules-transform and autosync` GitHub action will be triggered. We will not validate that action was successful.
```
Comment (jpower432) [inline docs/tutorials/authoring.md:37]:
```suggestion
Once the pull request has been merged the `Trestle-bot rules-transform and autosync` GitHub action will be triggered. We will now validate that action was successful.
```

File: docs/tutorials/authoring.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+```
+    "description": "",
+    "description": "Here is where detials should be added by the author",
+```
+
+You will also notice the `"last-modified"` timestamp has been updated.
+
+
+Congrats!  You've successfully authored a change by modifying a Markdown file and letting Trestle-bot sync those changes back to the OSCAL content.
```
Comment (jpower432) [inline docs/tutorials/authoring.md:56]:
```suggestion
Congrats!  You've successfully authored a change by modifying a Markdown file and letting trestle-bot sync those changes back to the OSCAL content.
```

**Review by jpower432 — COMMENTED** [review]
Nice work! I left a few suggestions.

**Review by jpower432 — APPROVED** [review]
LGTM

**Review by hbraswelrh — COMMENTED** [review]
This looks great! The details included are helpful for navigating the contents of the repository and provide context for GitHub Actions behavior.

**Pull request merged**